pg_search/src/api/aggregate.rs

```diff
@@ -50,3 +78,36 @@ pub fn aggregate(
         Ok(JsonB(serde_json::to_value(aggregate)?))
     }
 }
+
+/// Placeholder aggregate for `paradedb.agg()`.
```

Comment:
Oh, last (major) comment: everything has migrated to the `pdb` namespace, and this should too. So `pdb.agg`.